search.go

```diff
@@ -273,6 +273,7 @@ type SearchRequest struct {
 	Explain          bool              `json:"explain"`
 	Sort             search.SortOrder  `json:"sort"`
 	IncludeLocations bool              `json:"includeLocations"`
+	NoScore          bool              `json:"noScore"`
```

Comment:
Would it perhaps do good if this were a string instead, and just call it score; would need to default to "td/idf", and if one doesn't want to apply scoring, the option to enter in would be "none". This is so that should we introduce custom scoring at some point, we would just be able to re-use this setting rather than introducing a new one.